integration_tests/models/test_dateadd.sql

```diff
@@ -0,0 +1,51 @@
+with Input_data as (
+    select
+        '2023-11-12 13:14:15.678' as `testdate`,
+        null as `null_value`,
```

Comment:
This should be a datetime `NULL` value.

Reply:
Technically you are correct. We just removed the test for type tests from pm_utils. We can add it in the test and add Databricks support, but it is not compatible with Spark (no information_schema available). We discussed it and for now we will leave it this way, we will add a test if we run into an issue here.